app/forms/steps/other_info_form.rb

```diff
@@ -0,0 +1,28 @@
+require 'steps/base_form_object'
+
+module Steps
+  class OtherInfoForm < Steps::BaseFormObject
+    BOOLEAN_FIELDS = %i[concluded].freeze
+
+    attribute :other_info, :string
+    attribute :conclusion, :string
+
+    validates :other_info, presence: true
+    validates :conclusion, presence: true
```

Comment:
this should be another optional validation (I believe)..

so
```ruby
validates :conclusion, presence: true, if: :concluded
```

Reply:
thanks - will use in future